transform.go

```diff
@@ -48,15 +48,16 @@ func (expr mergeExpr) RootRefs() []FieldExpr {
 }
 
 type Transformer struct {
-	VarIdx          VariableID
-	BucketIdx       BucketID
-	RootExec        *ExecNode
-	RootTree        binTree
-	NodeMap         map[int]*ExecNode
-	ActiveExec      *ExecNode
-	ActiveBucketIdx BucketID
-	MaxDepth        int
-	CurDepth        int
+	VarIdx              VariableID
```

Comment:
These lines don't appear to actually be related to this change.

Reply:
Agreed. I'm not sure why they showed up in the diff.